skietbaan_fe/src/bootstrap/NavbarMenuStyle.css

```diff
@@ -104,12 +103,24 @@
   width: 26px;
 }
 
+.icon-same-dimensions:hover{
```

Comment:
I think the same can be achieved by declaring it like this:
```
.icon-same-dimensions:hover,
.profile-icon-grey:hover,
.notifications-icon-grey:hover {
  cursor: pointer;
}
```

Reply:
thanks i didnt know what was the correct way to do it